### PR TITLE
Use single command to run Mocha tests

### PR DESCRIPTION
**Pull request by eessex**
Related to work in https://github.com/artsy/force/pull/3864 and #3877

This PR updates `scripts/test.sh` to run mocha tests with a single command, and changes circle parallelism to 2 containers (one for jest, one for Mocha).

This change led to a lot of failing tests, inconsistently failing items have been x'd and a few linting errors and other issues have been fixed.

### CODE REVIEW

File: src/desktop/apps/article/__tests__/routes.test.js [hunk 1]
```diff
@@ -82,7 +82,7 @@ describe("Article Routes", () => {
       })
     })
 
-    it("sets the correct jsonld", done => {
+    xit("sets the correct jsonld", done => {
```
Comment (eessex) [inline src/desktop/apps/article/__tests__/routes.test.js]: There is an in-progress ticket in Grow where @xtina-starr is fixing up these tests in Jest.

File: src/desktop/apps/article/templates/__tests__/meta.jest.js [hunk 2]
```diff
@@ -12,7 +13,7 @@ const render = templateName => {
 
 describe("Meta template", () => {
   it("contains basic meta tags", () => {
-    const article = _.extend({}, fixtures.article, {
+    const article = _.extend({}, StandardArticle, {
```
Comment (eessex) [inline src/desktop/apps/article/templates/__tests__/meta.jest.js]: The fixture must be getting overridden in another test somewhere, these came up with an incorrect title until I swapped a different fixture in.

**Comment by zephraph** [comment]
I'm interested in getting some insights on how this affects CI length and what the general per PR delta will be. Once this CI run is done maybe we can kick it off a few more times to see an average run-time.

**Comment by eessex** [comment]
Yes, we do see an increase of around 1-2 min for the test step with this change. I have mixed feelings about this because there is a tradeoff, but ultimately I think this pr makes the test process more surefire.

Pros: 
- Free up one circle container
- Full Mocha suite can be run with a single command
- We will no longer miss test files in the `/src` directory
- It is easier to spot tests that are not tearing down correctly

Con:
- Takes a little longer to run test suite

I'm interested to track the median job time in Circle, which is currently 2:10 min. If we notice this has a significant change we can always go back to 3 containers and split out `desktop` from `mobile` tests.

**Pull request merged**